container/src/container/array_dict.py

```diff
@@ -9,196 +9,194 @@ def isiterable(obj):
         return False
 
 class array_dict:
```

Comment:
I did not changed anything in the 'array_dict' class ... as **not** showed by the whole diff, it seems that the differences are tabs converted into spaces...
The name was like this before my commit...